hawc/apps/epiv2/serializers.py

```diff
@@ -0,0 +1,157 @@
+from django.core.exceptions import ObjectDoesNotExist
+from rest_framework import serializers
+
+from ..common.serializers import FlexibleChoiceField, IdLookupMixin
+from ..epi.serializers import StudyPopulationCountrySerializer
+from ..study.serializers import StudySerializer
+from . import constants, models
+
+
+class AgeProfileSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = models.AgeProfile
+        fields = ["name"]
+
+    def to_internal_value(self, data):
+        if type(data) is str:
+            try:
+                age_profile = self.Meta.model.objects.get(name__iexact=data)
+                return age_profile
+            except ObjectDoesNotExist:
+                raise serializers.ValidationError(f"'{data}' is not a valid Age profile.")
+
+        return super().to_internal_value(data)
+
+
+class MeasurementTypeSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = models.MeasurementType
+        fields = ["description"]
+
+
+class ChemicalSerializer(serializers.ModelSerializer):
```

Comment:
see note below; probably best to delete this in the current PR being submitted (but keep in a separate branch for future work)